app/views/studies/samples/index.html.erb

```diff
@@ -7,7 +7,15 @@
   Study <%= @study.id %> &middot; Created on <%= @study.created_at.to_formatted_s(:long) %> &middot; <%= @study.state.capitalize %>
 </div>
 
+
+
 <% if @samples.size > 0 %>
+  <header>
+    <h3>Samples
+      <%= counter_badge @study.samples.count %>
+    </h3>
+  </header>
```

Comment:
Might the other paged benefit from a `count` badge too?

Reply:
The other ones have already the count badge. I have added here as it shows the total number of records.